aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/AddVersionCommand.java

```diff
@@ -81,23 +81,39 @@ public AddVersionCommand(RestApiService restApiService, JobsService jobsService,
 
     @CommandLine.Option(names = "--snapshot-name", paramLabel = "SNAPSHOT_NAME", description = "The name of the snapshot to generate")
     private String snapshotName;
+
     /**
      * Disable cloning previous version automatically.
      */
     @CommandLine.Option(names = {"--no-clone", "--no-rescan", "--new-configuration"},
-            description = "Enable this flag to create a new version without cloning the latest version configuration.",
-            defaultValue = "false")
+            description = "Enable this flag to create a new version without cloning the latest version configuration. Default: ${DEFAULT-VALUE}",
+            defaultValue = "false", fallbackValue = "true")
     private boolean disableClone = false;
+
+    /**
+     * Whether or not to clone previous version: backward compatibility
+     */
+    @CommandLine.Option(names = {"-c", "--clone", "--rescan", "--copy-previous-config"}
+            , description = "Clones the latest version configuration instead of creating a new application",
+            hidden = true, defaultValue = "false", fallbackValue = "true")
```

Comment:
why default value is false

Reply:
because disableClone = false was initialized with false representing the default value